backend/bot/modules/tosurnament/player.py

```diff
@@ -474,6 +476,8 @@ async def reschedule_handler(self, ctx, error):
             await self.send_reply(ctx, ctx.command.name, "past_deadline_end")
         elif isinstance(error, tosurnament.SameDate):
             await self.send_reply(ctx, ctx.command.name, "same_date")
+        elif isinstance(error, tosurnament.ImpossibleReschedule):
```

Comment:
this should not check if it is an instance of `ImpossibleReschedule` but of `pastNow`

Reply:
Oh my silly mistake... :(